### PR TITLE
Feature/fro97 litigation page UI

### PR DESCRIPTION
**Pull request by PatFawbertMills**
- UI for litigation
- Number of redirects at the next config level

### CODE REVIEW

File: frontend/.vscode/settings.json [hunk 1]
```diff
@@ -9,4 +9,5 @@
     "titleBar.activeBackground": "#006FD6",
     "titleBar.activeForeground": "#ffffff"
   },
-}
\ No newline at end of file
+  "scss.lint.unknownAtRules": "ignore"
```
Comment (PatFawbertMills) [inline frontend/.vscode/settings.json:12]: Just FYI this is because the Tailwind @ rules were causing VSCode warnings and errors

**Review by diversemix — APPROVED** [review]
Did you mean not to use the `AccordianItem` component you just created? - if so fine!

**Comment by PatFawbertMills** [comment]
> Did you mean not to use the `AccordianItem` component you just created? - if so fine!

In the `/litigation/[id].tsx` file you can see me using it. Github has hidden the file by default cause it's a long one.

**Review by joel-wright — APPROVED** [review]
LGTM - I see the redirect from the dummy litigation -> `/`

**Pull request merged**